lib/spack/spack/modules.py

```diff
@@ -527,8 +537,13 @@ class TclModule(EnvModule):
     path = canonicalize_path(
         _roots.get(name, join_path(spack.share_path, 'modules')))
 
+    def autoload_warner(self):
+        if self.verbose_autoload():
+            return 'puts stderr "Autoloading {module_file}"\n'
```

Comment:
The problem is that what you had before would leave the subsequent `load()` line doubly indented in the output file.  I'm fine with alternate solutions to that problem.